src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/locations/LocationsSynchronisationService.kt

```diff
@@ -146,3 +152,270 @@ private fun LocationsOffenderEvent.toTelemetryProperties(
 ) + (dpsLocationId?.let { mapOf("dpsLocationId" to it) } ?: emptyMap()) + (
   mappingFailed?.let { mapOf("mapping" to "initial-failure") } ?: emptyMap()
   )
+
```

Comment:
This section is basically a copy of the migration mappings but just using the separate NomisSyncLocationRequest type